docs/troubleshooting.md

```diff
@@ -449,3 +451,53 @@ Non-authoritative answer:
 Name:   bitnami.com
 Address: 50.17.235.25
 ```
+
+## Troubleshooting Elasticsearch
+
+### Elasticsearch crashloop under EKS
```

Comment:
Following the pattern used in the rest of the document where we first state the issue that's observed (i.e. Elasticsearch enters a crashloop) and then include a Troubleshooting section which would walk the user to confirm that the issue is due to the ulimits and then suggest using the `sed` command to resolve the issue.

Reply:
What about now?